lib/std/os/linux.zig

```diff
@@ -1216,6 +1216,10 @@ pub fn copy_file_range(fd_in: fd_t, off_in: ?*i64, fd_out: fd_t, off_out: ?*i64,
     );
 }
 
+pub fn bpf_syscall(cmd: bpf.Cmd, attr: *bpf.Attr, size: u32) usize {
```

Comment:
Here's a proposal for you:
 * make the bpf "bits" decl use screaming caps (in os/bits/linux.zig `pub const BPF = @import("linux/bpf.zig");`)
 * then this function can be called `bpf` instead of `bpf_syscall` to match the rest of the syscalls in this file.

Reply:
THANK YOU, I will get these changes in later today after work